script.js

```diff
@@ -1,53 +1,154 @@
+//trouble shoot reversed os message
+var totalMatches = 0;
+var timesPlayerWon = 0;
+var numberOfDraws = 0;
+var userName = ``;
+var message;
+var normal = `normal`;
+var korean = `korean`;
+var reversed = `reversed`;
+var gameMode = normal;
+var player = `player`;
+var currentWinner = `${userName}`;
+var myOutputValue;
 var main = function (input) {
+  var helpChoose = help();
   var elementfinal = elements();
-  myOutputValue = `The computer chose ${elementfinal} .
+  if (input == ``) {
+    input = helpChoose;
+  }
+  if (input == normal || input == korean || input == reversed) {
+    gameMode = input;
+    myOutputValue = `Your gamemode has been set to ${gameMode}. Please type 'scissors', 'paper' or 'stone' to play the game`;
+  }
+  customKorean = function (customPhrase) {
+    Message = `Thank you for playing ${userName}! The computer chose ${elementfinal} .
+You chose ${input}.
+<br>${customPhrase}.
+<br> Type in something other than "scissors", "paper","stone" ,"normal","reversed" or "korean" to set/change your username.
+<br>Now you can type "scissors" "paper" or "stone" to play another round!
+<br>You can also type in "normal", "reversed" or "korean" to change your game mode.(Your current game mode is ${gameMode})
+<br> If you would like the computer to play for you, leave the input box blank`;
+    return Message;
+  };
```

Comment:
Avoid declaring functions within another function. Only call functions within other function, similar to what you did with the help function